apps/etl-func/src/func.ts

```diff
@@ -34,6 +60,43 @@ const main = async (config: Config) => {
     methods: ["GET"],
     route: "health",
   });
+
+  const messageIngestion = async (documents: unknown[]) => {
+    const messagesToSend: MessageEvent[] = [];
+    for (const messageMetadataFromCosmosDB of documents) {
+      const messageMetadata = messageMetadataSchema.parse(
+        messageMetadataFromCosmosDB,
+      );
+
+      const message =
+        await messageAdapter.getMessageByMetadata(messageMetadata);
+
+      if (message !== undefined) {
+        const messageEvent = await getMessageEventFromMessage(
+          message,
+          PDVTokenizer,
+        );
+        messagesToSend.push(messageEvent);
+      } else {
+        logger.info(
+          `${messageMetadata.id} returned an undefined message event`,
+        );
+      }
+    }
+
+    await producer.publish(messagesToSend);
+  };
+
+  app.cosmosDB("messagesCosmosDBTrigger", {
+    connection: "COSMOSDB_URI",
+    containerName: config.messageCosmosDB.containerName,
+    createLeaseContainerIfNotExists: true,
+    databaseName: config.messageCosmosDB.databaseName,
+    handler: messageIngestion,
+    leaseContainerName: `${config.messageCosmosDB.containerName}-ingestion-lease`,
+    maxItemsPerInvocation: 30,
+    startFromBeginning: true,
```

Comment:
If we want to use `startFromBeginning: true`, then we should provide a filter in the business logic in order to process only from the date that we want

Reply:
Yes that's right. I'm using only this property right now to avoid possible problems with the dataset prepared for the test. 

We can choose between:
1. add the required parameter **startFromTime** only after the test succeeded.
2. use for the tests **startFromTime** parameter